imapclient/imapclient.py

```diff
@@ -109,8 +110,6 @@ class IMAPClient(object):
     calls if required.
 
     """
-
-    Error = imaplib.IMAP4.error
```

Comment:
I'm in 2 minds on this. I'm not fond of having the exceptions on the class as it's a little unconventional/surprising, but I'm also worried about breaking everyone's code.

Perhaps leave `Error` there but don't document it?